content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,90 @@
+| `permit-port-sharing`    | Default `false`. Use the default unless you are confident you understand the security implications.                                                                                |
+{{</table>}}
+
+### TLS configuration options
+{{<table caption="Address Configurations">}}
+| Configuration                  | Security hardening advice                                                                                                                                                                                                          |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `requestheader-client-ca-file` | Generally do not depend on authorization being already done for incoming requests. Always provide the root certificate bundle. This allows authorization to happen on each incoming request through `requestheader-allowed-names`. |
+| `tls-cipher-suites`            | Always provide a list of preferred cipher suites. This ensures encryption never happens with insecure cipher suites.                                                                                                               |
+{{</table>}}
+
+## Scheduling configurations for custom schedulers
+This section covers security hardening when using custom schedulers.
+The cluster administrator needs be careful with the plugins that use the `queueSort`, `filter`, or permit [extension points](https://github.com/docs/reference/scheduling/config/#extension-points).
+Scheduling happens in a series of stages that are exposed through the extension points.
+As a cluster administrator, you can enable plugins that define their own extension points.
+Doing so can affect the defined scheduling behaviors of the kube-scheduler in your cluster.
+
+When using a custom scheduler plugin extension points such as `queueSort`, `filter` and `permit` should be used with care.
+
+Exactly one plugin that uses the `queueSort` extension point can be enabled at a time. Any plugins that use `queueSort` should be scrutinized.
+
+Plugins that implement the `filter` extension point can potentially mark all nodes as unschedulable. This can bring scheduling of new pods to a halt.
+
+Plugins that implement the `permit` extension point  can prevent or delay the binding of a Pod. Such plugins should be thoroughly reviewed by the cluster administrator.
+
+When using a plugin that is not one of the [default plugins](https://kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins), consider disabling the `queueSort`, `filter` and `permit` extension points as follows:
```

Comment:
```suggestion
When using a plugin that is not one of the [default plugins](/docs/reference/scheduling/config/#scheduling-plugins), consider disabling the `queueSort`, `filter` and `permit` extension points as follows:
```

Reply:
Adding the `https://kubernetes.io` would always point to the main website when viewed on site preview or local runs. Would it better it always points to the version of the website that this viewed?

Reply:
Using a site-relative URL is better. Imagine how this page will look when the base URL is https://v1-31.docs.kubernetes.io/docs/concepts/security/hardening-guide/scheduler/ and the current release has moved on to v1.33.2

Would you want people to switch versions without noticing, or would you want to send them to https://v1-31.docs.kubernetes.io/docs/reference/scheduling/config/#scheduling-plugins ?

I'd definitely want the second option.

Reply:
So, I suggest adding refs using only the paths so Hugo can use the currently active host. So if you visit `https://v1.docs.kubernetes.io`, then you see the `/docs/reference/scheduling/config/#scheduling-plugins` from the active version, i.e., `v1.docs.kubernetes.io`.

Reply:
To do that, commit the suggestion from https://github.com/kubernetes/website/pull/45080#discussion_r1597533285